pkg/account/model.go

```diff
@@ -14,7 +19,24 @@ type Account struct {
 type AcountType string
 
 const (
-	ACCOUNT_TYPE_PATIENT AcountType = "patient"
-	ACCOUNT_TYPE_DOCTOR  AcountType = "doctor"
-	ACCOUNT_TYPE_ADMIN   AcountType = "admin"
+	patient AcountType = "patient"
+	doctor  AcountType = "doctor"
+	admin   AcountType = "admin"
 )
+
+type Doctor struct {
+	ID           uint `gorm:"primarykey"`
+	DepartmentID uint
+
+	AccountID uint
+	Cases     []cases.Case `gorm:"foreignkey:ID"`
```

Comment:
Doctors do not have cases. Any view on this? @PAN-Ziyue

Reply:
What about cases that a doctor responsible for?

Reply:
Well that's not in our design. Please delete this first, we may talk about this later.

Reply:
> Doctors do not have cases. Any view on this? @PAN-Ziyue

sorry, I don't get your point. And personally, I suggest speaking Chinese for better expression in issue and PR